closure/templates/test/BUILD

```diff
@@ -116,6 +128,18 @@ closure_js_test(
     ],
 )
 
+closure_js_binary(
+    name = "greeter_idom_fail_bin",
+    language = "ECMASCRIPT5_STRICT",
+    deps = [
+        ":greeter_idom_fail_lib",
+        "//closure/library",
+        "//closure/library:testing",
+    ],
+    pedantic = True,
+    internal_expect_failure = True,
```

Comment:
Thanks. As long as I know that is the intent I'll toss it on my list of projects that I can work on as time permits.